src/goto-instrument/contracts/contracts.cpp

```diff
@@ -395,22 +395,22 @@ void code_contractst::replace_old_parameter(
     const auto &parameter = to_old_expr(expr).expression();
 
     // TODO: generalize below
```

Comment:
May be remove this comment now, since this matches the assigns clause constraints? Or actually, may be mention: `TODO: Look into ID_ptrmember`?

Reply:
Remove it.

Reply:
I don't think the `ID_ptrmember` issue was fully resolved.